integration-tests/plugins/__tests__/product/admin/index.ts

```diff
@@ -576,10 +583,11 @@ describe("/admin/products", () => {
       expect(response?.data.product).toEqual(
         expect.objectContaining({
           id: toUpdateWithSalesChannels,
-          sales_channels: [
-            expect.objectContaining({ id: "channel-2" }),
-            expect.objectContaining({ id: "channel-3" }),
-          ],
+          // TODO: Introduce this in the sale channel PR
```

Comment:
@fPolic do you think this can be added to your PR?

Reply:
yes!